.github/workflows/build_and_test.yml

```diff
@@ -60,7 +60,7 @@ jobs:
             config-zstd: OFF
           - image: centos7
             python: python3.8
-            test-flags: --test-args="-s"
+            test-flags: --pcap-test-args="-s -x TestLoggerMultiThread"
```

Comment:
This is not directly related, but CentOS sometimes fails in this test, so we can disable it for now until we have time to look more into it

Reply:
Could be nice to have a comment in yaml file directly to explain why this test is disabled.

Reply:
Added a comment here: https://github.com/seladb/PcapPlusPlus/pull/1192/commits/90ed7e130d6b2ef660efd09b502dfa797d87783e